cmd/migration/main.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+// Here lives database migrations
+// Heroku runs this automagically before the server command (see Procfile)
+
+import (
+	"log"
+
+	"github.com/hex337/alex-koin-go/Config"
+	"github.com/hex337/alex-koin-go/Model"
+)
+
+func main() {
+
+	log.Println("Starting database migrations")
+	Config.GetEnvVars()
+	Config.DBOpen()
+
+	// TODO SELECT pg_try_advisory_lock(migration); and if f don't run migration as they are already happening
+	Config.DB.AutoMigrate(&Model.Transaction{}, &Model.User{}, &Model.Coin{})
```

Comment:
```suggestion
	Config.DB.AutoMigrate(&Model.User{}, &Model.Coin{}, &Model.Transaction{})
```

Reply:
Transactions needs to exist first of the FK constraints break. It does it struct by struct.

Reply:
When I was running this, it would barf because it tries to create the transactions table first, which has a fk on users, which didn't exist yet. This order worked though.